controller/controller.go

```diff
@@ -1439,7 +1439,7 @@ func (ctrl *ProvisionController) provisionClaimOperation(ctx context.Context, cl
 	if err := ctrl.volumeStore.StoreVolume(claim, volume); err != nil {
 		return ProvisioningFinished, err
 	}
-	if err = ctrl.volumes.Add(volume.Name); err != nil {
+	if err = ctrl.volumes.Add(volume); err != nil {
```

Comment:
What about changing this too? https://github.com/kubernetes-sigs/sig-storage-lib-external-provisioner/blob/master/controller/controller.go#L1520